pages/global-profile/content/index.js

```diff
@@ -0,0 +1,26 @@
+const { merge } = require('lodash');
+const profileContent = require('../../profile/read/content');
+const pilContent = require('../../task/read/content/pil');
+const tasklistContent = require('../../task/list/content');
+
+module.exports = merge({}, pilContent, profileContent, tasklistContent, {
+  establishment: {
+    link: 'About this establishment'
+  },
+  notifications: {
+    success: 'Changes saved!'
```

Comment:
Tiny thing - we could just fall back on the default here: 
https://github.com/UKHomeOffice/asl-pages/blob/master/pages/common/content/index.js#L178-L180

or at least keep them consistent (`!` vs `.`)